bumble/gatt.py

```diff
@@ -205,8 +205,16 @@ class Service(Attribute):
     '''
 
     uuid: UUID
+    characteristics: list[Characteristic]
```

Comment:
since we still support Python 3.8, we can't subscript builtin types, we have to use `List[xxx]` instead of `list[xxx]`. I notice that there's a couple other places in this file that already had the wrong case before this PR. Could you fix those as well?

Reply:
Done